test/commons/text/unicode.js

```diff
@@ -1,86 +1,94 @@
-      var actual = axe.commons.text.hasUnicode('ᴁ', {
+    it('returns true when given string is a phonetic extension', () => {
+      const actual = axe.commons.text.hasUnicode('ᴁ', {
         nonBmp: true
       });
       assert.isTrue(actual);
     });
 
-    it('returns true when given string is a combining diacritical marks supplement', function () {
-      var actual = axe.commons.text.hasUnicode('ᴁ', {
+    it('returns true when given string is a combining diacritical marks supplement', () => {
+      const actual = axe.commons.text.hasUnicode('ᴁ', {
         nonBmp: true
       });
       assert.isTrue(actual);
     });
 
-    it('returns true when given string is a currency symbols', function () {
-      var actual = axe.commons.text.hasUnicode('₨ 20000', {
+    it('returns true when given string is a currency symbols', () => {
+      const actual = axe.commons.text.hasUnicode('₨ 20000', {
         nonBmp: true
       });
       assert.isTrue(actual);
     });
 
-    it('returns true when given string has arrows', function () {
-      var actual = axe.commons.text.hasUnicode('← turn left', {
+    it('returns true when given string has arrows', () => {
+      const actual = axe.commons.text.hasUnicode('← turn left', {
         nonBmp: true
       });
       assert.isTrue(actual);
     });
 
-    it('returns true when given string has geometric shapes', function () {
-      var actual = axe.commons.text.hasUnicode('◓', {
+    it('returns true when given string has geometric shapes', () => {
+      const actual = axe.commons.text.hasUnicode('◓', {
         nonBmp: true
       });
       assert.isTrue(actual);
     });
 
-    it('returns true when given string has math operators', function () {
-      var actual = axe.commons.text.hasUnicode('√4 = 2', {
+    it('returns true when given string has math operators', () => {
+      const actual = axe.commons.text.hasUnicode('√4 = 2', {
         nonBmp: true
       });
       assert.isTrue(actual);
     });
 
-    it('returns true when given string has windings font', function () {
-      var actual = axe.commons.text.hasUnicode('▽', {
+    it('returns true when given string has windings font', () => {
+      const actual = axe.commons.text.hasUnicode('▽', {
         nonBmp: true
       });
       assert.isTrue(actual);
     });
 
-    it('returns true for a string with characters in supplementary private use area A', function () {
-      var actual = axe.commons.text.hasUnicode('\uDB80\uDFFE', {
+    it('returns true for a string with characters in supplementary private use area A', () => {
+      const actual = axe.commons.text.hasUnicode('\uDB80\uDFFE', {
+        nonBmp: true
+      });
+      assert.isTrue(actual);
+    });
+
+    it('returns true when given string has format unicode', () => {
+      // the front of this string contains a zero-width character
+      const actual = axe.commons.text.hasUnicode('⁩Hello World', {
```

Comment:
Instead of putting a comment and hoping this work, how about we use `\u` here?

```suggestion
      // zero-width spacer character U+200B
      const actual = axe.commons.text.hasUnicode('\u200BHello World', {
```